src/modules/RF69.cpp

```diff
@@ -87,8 +92,8 @@ int16_t RF69::begin(float freq, float br, float rxBw, float freqDev, int8_t powe
   }
 
   // default sync word values 0x2D01 is the same as the default in LowPowerLab RFM69 library
-  uint8_t syncWord[] = {0x2D, 0x01};
-  state = setSyncWord(syncWord, 2);
+  uint8_t syncWord[] = RF69_DEFAULT_SYNC_WORD;
```

Comment:
I'd rather keep the default 0x2D01 sync word, as per the comment.

Reply:
I don't get it, isn't `RF69_DEFAULT_SYNC_WORD` set to `{0x2D, 0x01}`?

Reply:
Ah, my mistake - I got confused with the changed default sync word in CC1101. There's no reason CC1101 should be kept compatible with LowPowerLab library, so it's fine as it is.